logger.go

```diff
@@ -63,6 +63,7 @@ func (l *MicroLogger) LogCtx(ctx context.Context, keyVals ...interface{}) {
 		if err != nil {
 			log.Printf("failed to log, reason: %#q", err.Error())
 		}
+		return
```

Comment:
https://github.com/giantswarm/micrologger/pull/38 broke it. Without the fix the test code panics. 

```
$ go test ./...
{"caller":"github.com/giantswarm/micrologger/logger_test.go:179","message":"test","time":"2020-03-20T16:09:12.75697+00:00"}
--- FAIL: Test_MicroLogger_LogCtx (0.00s)
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x116fd32]

goroutine 17 [running]:
testing.tRunner.func1(0xc000102b00)
	/Users/xh3b4sd/.gvm/gos/go1.13.3/src/testing/testing.go:874 +0x3a3
panic(0x11a5f00, 0x1338b50)
	/Users/xh3b4sd/.gvm/gos/go1.13.3/src/runtime/panic.go:679 +0x1b2
github.com/giantswarm/micrologger.(*MicroLogger).LogCtx(0xc000057130, 0x1219520, 0xc00001a0c8, 0xc00000ca60, 0x2, 0x2)
	/Users/xh3b4sd/go/src/github.com/giantswarm/micrologger/logger.go:72 +0x152
github.com/giantswarm/micrologger.Test_MicroLogger_LogCtx(0xc000102b00)
	/Users/xh3b4sd/go/src/github.com/giantswarm/micrologger/logger_test.go:179 +0x132
testing.tRunner(0xc000102b00, 0x11e69a0)
	/Users/xh3b4sd/.gvm/gos/go1.13.3/src/testing/testing.go:909 +0xc9
created by testing.(*T).Run
	/Users/xh3b4sd/.gvm/gos/go1.13.3/src/testing/testing.go:960 +0x350
FAIL	github.com/giantswarm/micrologger	0.241s
ok  	github.com/giantswarm/micrologger/loggermeta	(cached)
ok  	github.com/giantswarm/micrologger/microloggertest	(cached)
FAIL
```